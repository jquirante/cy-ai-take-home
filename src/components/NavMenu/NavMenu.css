.logo {
    display: inline-block;
    position: relative;
    left: 150px;
    font-size: 18px;
}