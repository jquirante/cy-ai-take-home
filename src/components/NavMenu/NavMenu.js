import React from 'react';
import './NavMenu.css';

const NavMenu = () => {
    return(
        <nav>
            <div className="nav-wrapper grey darken-4">
                <div className="logo">CyHub</div>
            </div>
        </nav>
    )
}

export default NavMenu;