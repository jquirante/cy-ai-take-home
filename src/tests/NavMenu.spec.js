import React from "react";
import NavMenu from '../components/NavMenu/NavMenu';
import { shallow } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

describe("NavMenu component", () => {
  Enzyme.configure({ adapter: new Adapter() });
  test("Matches the snapshot", () => {
    const navMenu = shallow(<NavMenu />);
    expect(navMenu).toMatchSnapshot();
  });
});
