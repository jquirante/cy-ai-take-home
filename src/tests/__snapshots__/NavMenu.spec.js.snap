// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavMenu component Matches the snapshot 1`] = `ShallowWrapper {}`;
